go/porcelain/deploy.go

```diff
@@ -513,6 +525,28 @@ func (n *Netlify) uploadFile(ctx context.Context, d *models.Deploy, f *FileBundl
 	}
 }
 
+func createFileBundle(rel, path string) (*FileBundle, error) {
```

Comment:
Abstracted into separate function so that it can be used by both `walk` and `walkEdgeHandlers`.